### PR TITLE
[zh] Fix markdown error

### LINKED ISSUE #31153: 在 Linux 系统中安装并设置 kubectl markdown 渲染错误
image

### PR DESCRIPTION
**Pull request by tengqm**
closes #31153

### CODE REVIEW

File: zhdiff.txt [hunk 1]
```diff
@@ -0,0 +1,423 @@
+2	2	content/en/includes/task-tutorial-prereqs.md
```
Comment (shuaijinchao) [inline zhdiff.txt:1]: this file should not appear in the project.

Comment (tengqm) [reply zhdiff.txt]: oh, my bad.

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**